src/libraries/Microsoft.PowerFx.Core/Public/IRuntimeConnectorContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net.Http;
+using System.Text;
+
+namespace Microsoft.PowerFx
+{
+    public interface IRuntimeConnectorContext
```

Comment:
>IRuntimeConnectorContext [](http://example.com/codeflow?start=21&length=24)

Let's discuss this 

My concerns...
1. does not belong in Core.  Let's find a way to move up the stack to interpreter. 
2.  Maybe should be a class so we can add to it later without breaking?

3. Mix of responsibilities? Http and Culture are a strange combination.  Perhaps overlap with FormattingInfo

Reply:
1 - I'm not sure of the motivation / the root of your concern. We have to find a way to call connector methods and use a contract for their runtime requirements
2- A class is less convenient than an interface for the callers. Here they just have to implement it and we can extended it easily if we need to
3 - There is no link between these items except they are used using during function execution. FormattingInfo isn't a so good idea as it's internal and we come to duplicated items in IServiceProvider